core-web/libs/edit-content-bridge/src/lib/bridges/angular-form-bridge.ts

```diff
@@ -49,31 +61,75 @@ export class AngularFormBridge implements FormBridge {
 
     /**
      * Subscribes to field changes in the Angular form.
+     * Supports multiple callbacks per field.
      *
      * @param fieldId - The ID of the field to subscribe to.
      * @param callback - The callback function to execute when the field changes.
+     * @returns A function to unsubscribe this specific callback.
      */
-    onChangeField(fieldId: string, callback: (value: FormFieldValue) => void): void {
-        const existingSubscription = this.subscriptions.get(fieldId);
-        if (existingSubscription) {
-            existingSubscription.unsubscribe();
-            this.subscriptions.delete(fieldId);
+    onChangeField(fieldId: string, callback: (value: FormFieldValue) => void): () => void {
+        const callbackId = `callback_${++this.callbackIdCounter}`;
+        const fieldCallback: FieldCallback = { id: callbackId, callback };
+
+        let fieldSubscription = this.fieldSubscriptions.get(fieldId);
+
+        if (!fieldSubscription) {
+            // Create new subscription for this field
+            const subscription = this.form.get(fieldId)?.valueChanges.subscribe((value) => {
+                const currentFieldSubscription = this.fieldSubscriptions.get(fieldId);
+                if (currentFieldSubscription) {
+                    // Execute all callbacks for this field
+                    currentFieldSubscription.callbacks.forEach(({ callback: cb }) => {
+                        this.zone.run(() => cb(value));
+                    });
+                }
+            });
+
+            if (subscription) {
+                fieldSubscription = {
+                    subscription,
+                    callbacks: [fieldCallback]
+                };
+                this.fieldSubscriptions.set(fieldId, fieldSubscription);
+            }
+        } else {
+            // Add callback to existing subscription
+            fieldSubscription.callbacks.push(fieldCallback);
```

Comment:
Aren't we subscribing here a bunch of time.

Reply:
This will reuse a subscription or create one if it not exist.